container/Exchange.js

```diff
@@ -28,19 +44,17 @@ export default () => {
 
     return () => {
       cancelled = true
+      setTranslation(undefined)
```

Comment:
I think this got added by mistake again?

Reply:
ughhhh this must be when I reverted the commit holdup